maddie/experimentation/exercise_1.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+import numpy as np
+from scipy.stats import normaltest
+import matplotlib.pyplot as plt
+from matplotlib.patches import Rectangle
+import pandas as pd
+
+n_points = 100
+
+def generate_one_realisation():
+    return np.random.beta(1.8, 1, size=n_points)
+
+def infer(sample):
+    return np.mean(sample)
+
+data = [infer(generate_one_realisation()) for _ in range(100)]
+
+stat, p = normaltest(data)
+print('stat=%.3f, p=%.3f'% (stat, p))
+if p > 0.1:
+    print ('Null hypothesis accepted')
```

Comment:
You never accept the null hypothesis. You fail to reject the null.

This is all about falsification. You can have enough data to falsify a hypothesis, but you'll never have enough to proof that it's true.